index.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar" class="nav_menu">
+      <ul class="nav-items">
+        <li><a href="#welcome-section"> About me </a></li>
+        <li><a href="#projects"> My Projects </a></li>
+        <li><a href="#contact"> Contact me </a></li>
+      </ul>
+    </nav>
+
+    <section id="welcome-section" class="intro-sect">
```

Comment:
Please notice that, as said in the previous comment, only one attribute is enough to refer to an element. In this case, since the id attribute is used to hyperlink the section, avoid using the class attribute too.

Reply:
- Outside the scope of this review, just to clarify and future references, the classes are better to work with CSS, [check this article](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) for more info about it.